packages/turf-helpers/package.json

```diff
@@ -69,7 +68,7 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
-    "deep-equal": "^2.2.3",
+    "dequal": "^2.0.3",
```

Comment:
Did you consider `fast-deep-equal` at all? I hadn't heard of `dequal` but it seems plenty popular. I didn't compare final bundle sizes with both, but they're both much smaller than `deep-equals`

Reply:
they both seem pretty good. I don't really have much of a preference for one vs the other